lib/components/Application.js

```diff
@@ -1,10 +1,13 @@
 import React, { Component } from 'react';
+// const HappyFace = require('../../public/img/happy-face.png');
```

Comment:
I copied the loaders section from the old repo and it's working now.  no idea what the issue was exactly with the way you had it configured but I'm able to see both the happy face and the celebrating guy now.  pull down the repo and make sure it's loading how you were wanting it to.  I think this is ready to merge now.